api/src/main/java/io/opentelemetry/metrics/Counter.java

```diff
@@ -37,6 +51,9 @@
      * @param monotonic {@code true} only positive values are expected.
      * @return this.
      */
-    B setMonotonic(boolean monotonic);
+    Builder setMonotonic(boolean monotonic);
+
+    @Override
+    Counter<?> build();
```

Comment:
Having a wildcard type in a public API feels very user-hostile. What would the user do with this?  Should this return `Counter<H>`?